library/templates/dcp_pops.yml

```diff
@@ -0,0 +1,33 @@
+dataset:
+  name: &name dcp_pops
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: library/tmp/POPS_Open_Data.GeoJSON
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
```

Comment:
geometry should be point or somthing?

Reply:
Ok. We'd previously loaded it with no geom.